@xen-orchestra/web-core/lib/locales/en.json

```diff
@@ -53,6 +53,7 @@
   "no-data": "No data",
   "object-not-found": "Object {id} can't be found…",
   "other": "Other",
+  "page-not-found": "This page is not to be found…",
```

Comment:
I agree with your comment. However, we will keep the current wording as it is used in the design system.
@clemencebx Any thoughts?
Since it is not a big change, I will merge this PR, and we could update the wording later.